packages/nuscr/nuscr.2.0.0/opam

```diff
@@ -11,6 +11,7 @@ license: "GPL-3.0-or-later"
 homepage: "https://nuscr.dev/"
 doc: "https://nuscr.dev/nuscr/docs/"
 bug-reports: "https://github.com/nuscr/nuscr/issues"
+available: [ os != "freebsd" ]
```

Comment:
```suggestion
```

Reply:
Not sure why it fails, but it compiles fine and users may want to try it out and help fixing it